### PR TITLE
Run of river temp

### PR DESCRIPTION
**Pull request by MaddeeRubenson**
incorporates HEC5Q results from Run of River Calsim and adds additional air/temp modeling for missing tribs

### CODE REVIEW

File: data-raw/knights_landing/knights_landing_modeling.R [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+library(dataRetrieval)
+library(CDECRetrieve)
+library(forcats)
+library(rnoaa)
+library(caret)
+
+# hourly water temperature data on VON (fahrenheit)
+#07/29/1998 to present
+
+CDECRetrieve::cdec_stations("VON")
```
Comment (ErinCain) [inline data-raw/knights_landing/knights_landing_modeling.R]: What are the time serries on these stations. We should confirm that time they do not have coverage in the 1979 - 2000 period - if they do we can just use gage data directly and interpolate missing months.

Comment (ErinCain) [reply data-raw/knights_landing/knights_landing_modeling.R]: Lets talk through this tomorrow

Comment (MaddeeRubenson) [reply data-raw/knights_landing/knights_landing_modeling.R]: There is a USGS gage at Knights Landing that has water temperature data but only for a year from 1995 to 1996 https://waterdata.usgs.gov/ca/nwis/inventory/?site_no=11391100

File: NEWS.md [hunk 2]
```diff
@@ -1,12 +1,14 @@
 # DSMtemperature 2.0
 
-The 2022 update to DSMtemperature includes the following changes:
+The Reorienting to Recovery and 2022 update to DSMtemperature includes the following changes:
```
Comment (ErinCain) [inline NEWS.md:3]: Lets rebrand this as V1.0 for R2R. Please update to remove the old stuff and describe using this data with the R2R models

File: NEWS.md [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
 
 * **CALSIM updates**. Updated `stream_temperature` and `degree_days` datasets include new 2019 biop flows. The result of
 doing so means accessing each temperature dataset must be done using the following format: 
-`stream_temperature$biop_2008_2009` and `stream_temperature$biop_itp_2018_2019`.
+`stream_temperature$biop_2008_2009`, `stream_temperature$biop_itp_2018_2019`, and `run_of_river`.
 
-* **Modeling updates**. The Hec5q models associated with the 2019 biop no longer support the San Joaquin river and its tributaries. We had to do additional temperature modeling to make up for these data gaps. Data for Merced River, Stanislaus River, Tuolumne River, and the San Joaquin River is now modeled using an air to water linear regression. 
+* **2022 BiOp: Modeling updates:** The Hec5q models associated with the 2019 biop no longer support the San Joaquin river and its tributaries. We had to do additional temperature modeling to make up for these data gaps. Data for Merced River, Stanislaus River, Tuolumne River, and the San Joaquin River is now modeled using an air to water linear regression.
```
Comment (ErinCain) [inline NEWS.md]: what is the 2022 biop? this is not a run that exists. This label does not make sense

File: R/data.R [hunk 4]
```diff
@@ -59,9 +60,20 @@
 #'
 #' @section Watershed Modeling Details:
 #' \itemize{
-#'    \item \strong{Upper Sacramento River} HEC-5Q model output at COTTONWOOD CR in SSJB_SAC_Reference_062315/SAC/SAC_CL_TEMP.DSS
+#'    \item \strong{Upper Sacramento River}
+#'      \itemize{
+#'        \item{biop_2008_2009: HEC-5Q model output at COTTONWOOD CR in SSJB_SAC_Reference_062315/SAC/SAC_CL_TEMP.DSS}
+#'        \item{biop_itp_2018_2019: HEC-5Q model output at COTTONWOOD CR in SSJB_SAC_Reference_062315/SAC/SAC_CL_TEMP.DSS}
```
Comment (ErinCain) [inline R/data.R]: The filepath would no longer be the same for this version. Please remove filepath

File: R/data.R [hunk 5]
```diff
@@ -59,9 +60,20 @@
...
+#'        \item{run_of_river: Estimated mean monthly water temperature from a linear model fitted with water temperature data from CDEC Gage ID: \href{# https://cdec.water.ca.gov/dynamicapp/staMeta?station_id=jlf}{JLF} and air temperature data from NOAA CDO Station Id: \href{https://www.ncdc.noaa.gov/cdo-web/datasets/GSOM/stations/GHCND:USW00024216/detail}{USW00024216}}
+#'    }
+#'
 #'    \item \strong{Antelope Creek} Regression method was fitted to data from in-river temperature gauges
-#'    \item \strong{Battle Creek} HEC-5Q model output at BATTLE CR in SSJB_SAC_Reference_062315/SAC/SAC_CL_TEMP.DSS
+#'    \item \strong{Battle Creek}
+#'      \itemize{
+#'        \item{biop_2008_2009: HEC-5Q model output at BATTLE CR in SSJB_SAC_Reference_062315/SAC/SAC_CL_TEMP.DSS}
+#'        \item{biop_itp_2018_2019: HEC-5Q model output at BATTLE CR in SSJB_SAC_Reference_062315/SAC/SAC_CL_TEMP.DSS}
```
Comment (ErinCain) [inline R/data.R]: same comment here. remove file path

File: R/data.R [hunk 6]
```diff
@@ -74,13 +86,23 @@
 #'    \item \strong{Paynes Creek} Regression method was fitted to data from Antelope River temperature gauges
 #'    \item \strong{Stony Creek} HEC-5Q model  output at STONY CR in SSJB_SAC_Reference_062315/SAC/SAC_CL_TEMP.DSS
 #'    \item \strong{Thomes Creek} HEC-5Q model output at THOMES CR in SSJB_SAC_Reference_062315/SAC/SAC_CL_TEMP.DSS
-#'    \item \strong{Upper-mid Sacramento River} HEC-5Q model output at STONY CREEK in SSJB_SAC_Reference_062315/SAC/SAC_CL_TEMP.DSS
-#'    \item \strong{Sutter Bypass} Tisdale Rotary Screw Trap montly mean water temperature 2011-2017
+#'    \item \strong{Upper-mid Sacramento River}
+#'     \itemize{
+#'       \item{biop_2008_2009: HEC-5Q model output at STONY CREEK in SSJB_SAC_Reference_062315/SAC/SAC_CL_TEMP.DSS}
+#'        \item{biop_itp_2018_2019: HEC-5Q model output at STONY CREEK in SSJB_SAC_Reference_062315/SAC/SAC_CL_TEMP.DSS}
```
Comment (ErinCain) [inline R/data.R]: and here

File: R/data.R [hunk 7]
```diff
@@ -74,13 +86,23 @@
...
 #'    \item \strong{Bear River} Regression method was fitted to data from Deer Creek temperature gauges
 #'    \item \strong{Feather River} Regression method was fitted to data from Deer Creek temperature gauges
 #'    \item \strong{Yuba River} Estimated mean monthly water temperature from a linear model fitted with water temperature data from USGS Gage ID: \href{https://waterdata.usgs.gov/nwis/inventory?agency_code=USGS&site_no=11421000}{11421000} and air temperature data from NOAA CDO Station Id: \href{https://www.ncdc.noaa.gov/cdo-web/datasets/GSOM/stations/GHCND:USW00024216/detail}{USW00024216}
-#'    \item \strong{Lower-mid Sacramento River} HEC-5Q model output at KNIGHTS LDG in SSJB_SAC_Reference_062315/SAC/SAC_CL_TEMP.DSS
-#'    \item \strong{Yolo Bypass} Knights Landing Rotary Screw Trap montly mean water temperature 2004-2018
+#'    \item \strong{Lower-mid Sacramento River}
+#'      \itemize{
+#'        \item{biop_2008_2009: HEC-5Q model output at KNIGHTS LDG in SSJB_SAC_Reference_062315/SAC/SAC_CL_TEMP.DSS}
+#'        \item{biop_itp_2018_2019: HEC-5Q model output at KNIGHTS LDG in SSJB_SAC_Reference_062315/SAC/SAC_CL_TEMP.DSS}
```
Comment (ErinCain) [inline R/data.R]: same here

**Review by ErinCain — COMMENTED** [review]
Please make the described changes and then I will merge in

File: R/data.R (hunk 4 above)
Comment (MaddeeRubenson) [reply R/data.R]: What would it be? Should I just reference the HEC 5Q model output?

Comment (ErinCain) [reply R/data.R]: yeah that is fine

File: NEWS.md (hunk 3 above)
Comment (MaddeeRubenson) [reply NEWS.md]: derp - updated

File: R/data.R (hunk 4 above)
Comment (MaddeeRubenson) [reply R/data.R]: updated

File: R/data.R (hunk 6 above)
Comment (MaddeeRubenson) [reply R/data.R]: updated

File: R/data.R (hunk 7 above)
Comment (MaddeeRubenson) [reply R/data.R]: updated

**Review by ErinCain — APPROVED** [review]

**Pull request merged**